### DISCUSSION TITLE
Gerrit connector

### DISCUSSION THREAD

**Discussion by gnl42** [eclipse-mylyn/org.eclipse.mylyn#511] [General]
Are we still supporting the Gerrit connector? Given that the code is written against version 2.12.0 (EOL) and the current version is 3.9

**Comment by ruspl-afed** [comment]
I would say we still have it in our source base, but it needs to be ported on more recent 3rd party libs to be re-enabled in SimRel.
Since EF has no Gerrit service in its infra this topic has lower priority than others.
We may consider some "open" instance of Gerrit to understand which version to support (if any).

**Comment by merks** [comment]
I think the EGit folks use Gerrit.  

@msohn 

I recall when redesigning Orbit that some of the 3rd party dependencies were super old and looked to be unmaintained.  But this is a few years ago now...

**Comment by msohn** [comment]
Yes, JGit and EGit use Gerrit. We use [GerritHub](https://gerrithub.io/) for code review, submitted changes are mirrored to GitHub.
See the [JGit and EGit contributor guide](https://github.com/eclipse-egit/egit/wiki/Contributor-Guide#contributing-patches).
GerritHub is a free Gerrit service hosted by [GerritForge](https://www.gerritforge.com/).

**Comment by gnl42** [comment]
Removed by #1178
